hooks/useFriends.ts

```diff
@@ -24,6 +25,7 @@ export const useFriends = () => {
   }, [friends, availableFriends])
 
   const hasFriends = useMemo(() => {
+    if(isLoading) return true
```

Comment:
why are we setting hasFriends to true while still loading?

Reply:
There is a split second where the noFriends screen still shows,